packages/appstore/src/components/modals/wallets-upgrade-modal/wallets-upgrade-modal.tsx

```diff
@@ -14,11 +14,9 @@ const WalletsUpgradeModal = observer(() => {
     const { is_eligible, startMigration } = useWalletMigration();
     const isWalletMigrationModalClosed = localStorage.getItem('is_wallet_migration_modal_closed');
     const [modalOpen, setModalOpen] = React.useState(!isWalletMigrationModalClosed);
-    const [iframeError, setIframeError] = React.useState(false);
 
     const iframeSrc =
         'https://customer-hhvo3ceuqt00w8g8.cloudflarestream.com/7ed5dfd27cdd290e088be8e91201a7f1/iframe?autoplay=true&preload=auto';
```

Comment:
small nit but maybe this const could be renamed as iframe is not being used anymore. And would there be a security issue with having the video url public or should it be saved as an env variable?

Reply:
I've removed the url and instead replace it with it's unique id.